app-admin/pinkybar/metadata.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="project">
```

Comment:
Proxy-maint always comes last, and please fill in the name. As I said, you've got a copy-paste'ish sample on the wiki.